overlord/snapstate/autorefresh_gating.go

```diff
@@ -143,6 +144,33 @@ func holdDurationLeft(now time.Time, lastRefresh, firstHeld time.Time, maxDurati
 	return d2
 }
 
+// HoldRefreshes holds the snaps with "system" as the gating snap.
```

Comment:
looking at the tests made me think about this function name, it looks very similar to HoldRefresh but actually it's quite different, because holding by system has different properties. Maybe it should be called HoldRefreshesBySystem? and the doc comment should explain how holdTime works, and that the hold can be arbitrary? also that it is used to implement holding by the sys admin.  Test names would also need to change to include the new function name in their name